pkg/util/test/util.go

```diff
@@ -1061,6 +1069,64 @@ func validateStorageClusterPods(
 	return nil
 }
 
+// validateDmthinOnPxNodes greps for dmthin in the /etc/pwx/config.json on each PX pods
+// and makes sure its there, if dmthin misc-args annotation is found
+func validateDmthinOnPxNodes(cluster *corev1.StorageCluster) error {
+	listOptions := map[string]string{"name": "portworx"}
+	cmd := "cat /etc/pwx/config.json | grep dmthin"
+
+	if cluster.Annotations != nil {
```

Comment:
nit: there's no need to check nil, it's valid to check if nil map contains some key.